src/Tribe/Main.php

```diff
@@ -566,44 +566,49 @@ public function should_load_common_admin_css() {
 	}
 
 	/**
-	 * A Helper method to load text domain
-	 * First it tries to load the wp-content/languages translation then if falls to the try to load $dir language files.
+	 * A helper method to load text domain.
+	 * First, it tries to load language files from a custom folder when set. If it fails it falls back to load from $dir.
 	 *
 	 * @since  4.0.1 Introduced.
 	 * @since  4.2   Included $domain and $dir params.
+	 * @since  TBD   Fix text domain loading logic after changes in WordPress 6.7.
 	 *
 	 * @param string       $domain The text domain that will be loaded.
 	 * @param string|false $dir    What directory should be used to try to load if the default doesn't work.
 	 *
 	 * @return bool  If it was able to load the text domain.
```

Comment:
```suggestion
	 * @return bool  If it was able to register the text domain.
```
To whit: `load_plugin_textdomain` now just registers the path and does not handle _laoding_ it. We retrun true if the domain is already loaded, or if we successfully _register_ the path